### PR TITLE
Fix eslint issues in account-lib

### PR DESCRIPTION
**Pull request by silalvado**

### CODE REVIEW

File: modules/account-lib/src/coin/celo/transactionBuilder.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class TransactionBuilder extends Eth.TransactionBuilder {
     return walletSimpleByteCode + resultEncodedParameters;
   }
 
-  //region Stake methods
+  // region Stake methods
```
Comment (marktoda) [inline modules/account-lib/src/coin/celo/transactionBuilder.ts:92]: @twtaylorbitgo says this isn't supposed to be here but I dunno :shrug:

Comment (gonzaalvarez70) [reply modules/account-lib/src/coin/celo/transactionBuilder.ts:92]: Do you mean it should be in another line, or we should not add the region at all? @twtaylorbitgo

File: modules/account-lib/.eslintrc.js [hunk 2]
```diff
@@ -20,6 +20,8 @@ module.exports =  {
     'object-curly-spacing': ['error', 'always'],
     '@typescript-eslint/camelcase': 'warn',
     '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    "@typescript-eslint/explicit-member-accessibility": "off",
```
[the remark this answers is not in the document]
Comment (silalvado) [reply modules/account-lib/.eslintrc.js]: Done. Thanks!

File: modules/account-lib/src/coin/celo/transactionBuilder.ts (hunk 1 above)
Comment (twtaylorbitgo) [reply modules/account-lib/src/coin/celo/transactionBuilder.ts:92]: Resolved this via private conversation

File: modules/account-lib/.gitignore [hunk 3]
```diff
@@ -10,3 +10,4 @@ package/
 coverage.lcov
 resources/trx/protobuf/*.d.ts
 resources/trx/protobuf/*.js
+/package-lock.json
```
Comment (twtaylorbitgo) [inline modules/account-lib/.gitignore:13]: sorry - why did we remove this?

Comment (silalvado) [reply modules/account-lib/.gitignore]: Removed. Maybe we can discuss it later.

**Pull request merged**